dispatcher/Pipelines/Integration/POST/Tableland/load_project.ts

```diff
@@ -29,6 +29,91 @@ const client = createThirdwebClient({
     clientId: process.env.NEXT_PUBLIC_THIRDWEB_CLIENT_ID as string,
 });
 
+const discordToEthAddress = {
```

Comment:
this could be tough to maintain over time w/ new addresses. Wish there were a way to get this data from the bot we use to unlock access in the discord